torchsynth/module.py

```diff
@@ -622,15 +622,18 @@ class Noise(SynthModule):
 
     # Do we really want deterministic noise within each batch?
     # https://github.com/turian/torchsynth/issues/250
-    noise_batch_size: int = 64
+    noise_batch_size: int = BATCH_SIZE_FOR_REPRODUCIBILITY
     # Unfortunately, Final is not supported until Python 3.8
-    # noise_batch_size: Final[int] = 64
+    # noise_batch_size: Final[int] = BATCH_SIZE_FOR_REPRODUCIBILITY
 
     def __init__(self, synthconfig: SynthConfig, seed: int, **kwargs):
         super().__init__(synthconfig, **kwargs)
 
         # https://github.com/turian/torchsynth/issues/255
-        assert self.batch_size == 64 or not self.synthconfig.reproducible
+        assert (
+            self.batch_size == BATCH_SIZE_FOR_REPRODUCIBILITY
+            or not self.synthconfig.reproducible
+        )
```

Comment:
I have a slight preference for raising an error with a message like we do in config here and below